qa/objectstore/bluestore.yaml

```diff
@@ -20,3 +20,21 @@ overrides:
         osd failsafe full ratio: .95
 # this doesn't work with failures bc the log writes are not atomic across the two backends
 #        bluestore bluefs env mirror: true
+  ceph-deploy:
+    fs: xfs
+    bluestore: yes
```

Comment:
@vasukulkarni , why would adding ceph-deploy overrides to filestore-xfs.yaml increase the job count?
all it would do is make sure, we also have test coverage for ceph-deploy with filestore: true, right?